soh/soh/Enhancements/randomizer/location_access/overworld/gerudo_valley.cpp

```diff
@@ -17,6 +17,7 @@ void RegionTable_Init_GerudoValley() {
         Entrance(RR_GV_CRATE_LEDGE,   []{return logic->IsChild || logic->CanUse(RG_LONGSHOT);}),
         Entrance(RR_GV_GROTTO_LEDGE,  []{return true;}),
         Entrance(RR_GV_FORTRESS_SIDE, []{return (logic->IsAdult && (logic->CanUse(RG_EPONA) || logic->CanUse(RG_LONGSHOT) || ctx->GetOption(RSK_GERUDO_FORTRESS).Is(RO_GF_CARPENTERS_FREE) || logic->CarpenterRescue)) || (logic->IsChild && logic->CanUse(RG_HOOKSHOT));}),
+        Entrance(RR_LAKE_HYLIA,       []{return logic->IsChild || logic->HasItem(RG_BRONZE_SCALE);}), //can use cucco as child
```

Comment:
```suggestion
        Entrance(RR_LAKE_HYLIA,       []{return logic->IsChild;}), //can use cucco as child
```
could leave the other entrance logic paths to let adult find their way to lake hylia with bronze scale